### PR TITLE
Update mentions of iOS to Swift

### PR DESCRIPTION
**Pull request by szekelyzol**
Changes are for [this Asana task](https://app.asana.com/0/1204307844224163/1205701294495344/f), as part of [this parent task](https://app.asana.com/0/1204307844224163/1205701152598729/f).

**Summary**: according to the parent task, I've renamed iOS to Swift in all instances for the Swift API client, Live stream lib, Uploader SDK, and Player analytics SDK.

- any page where we reference iOS in the list of offered clients / SDKs / libs
- `navigation.yaml` for [API client libraries](https://docs.api.video/sdks/api-clients)
- URLs in `templates\documentation\sdks\player\apivideo-swift-player.md`
- URLs in `templates\documentation\sdks\player\apivideo-swift-player-analytics.md`
- URLs and titles in `templates\documentation\sdks\vod\apivideo-ios-uploader.md`
- Filename and `navigation.yaml` reference of `templates\documentation\sdks\vod\apivideo-ios-uploader.md`
- URLs in `templates\documentation\sdks\vod\apivideo-react-native-uploader.md`
- Redirect to `/sdks/vod/apivideo-swift-uploader` in `templates\documentation\doctave.yaml`
- URLs in `templates\documentation\vod\upload-a-video-regular-upload.md`
- URLs in `templates\documentation\vod\upload-from-source.md`
- URLs in `templates\documentation\vod\upload-your-first-video.md`

### CODE REVIEW

File: templates/documentation/reference/video-source-already-uploaded.md [hunk 1]
```diff
@@ -25,6 +25,6 @@ You can cut down on mistakes by using one of our clients. We offer clients for o
 - [Go](../sdks/api-clients/apivideo-go-client.md)
 - [C#](../sdks/api-clients/apivideo-csharp-client.md)
 - [Java](../sdks/api-clients/apivideo-java-client.md)
-- [iOS](../sdks/api-clients/apivideo-swift5-client.md)
+- [iSwiftOS](../sdks/api-clients/apivideo-swift5-client.md)
```
Comment (ThibaultBee) [inline templates/documentation/reference/video-source-already-uploaded.md:28]: It seems to be copy/paste issue here

**Comment by ThibaultBee** [comment]
Have you changed the iOS thumbnail to the Swift thumbnail?

File: templates/documentation/reference/video-source-already-uploaded.md (hunk 1 above)
Comment (szekelyzol) [reply templates/documentation/reference/video-source-already-uploaded.md:28]: Ah yes... the brand new platform from Apple, iSwiftOS! 😬 Let me fix it, along with the Swift icons. Great catch!

**Review by szekelyzol — COMMENTED** [review]
@ThibaultBee 2 fixes added, please check 🙏

**Review by ThibaultBee — APPROVED** [review]

**Pull request merged**